MediaBrowser.WebDashboard/dashboard-ui/strings/ar.json

```diff
@@ -1261,7 +1261,7 @@
     "LabelPlayMethodDirectPlay": "\u0627\u0644\u062a\u0634\u063a\u064a\u0644 \u0627\u0644\u0645\u0628\u0627\u0634\u0631",
     "LabelAudioCodec": "\u0645\u0642\u0637\u0639 \u0627\u0644\u0635\u0648\u062a: {0}",
     "LabelVideoCodec": "\u0645\u0642\u0637\u0639 \u0627\u0644\u0641\u064a\u062f\u064a\u0648: {0}",
-    "LabelLocalAccessUrl": "In-Home (LAN) access: {0}",
+    "LabelLocalAccessUrl": "LAN address: {0}",
     "LabelRemoteAccessUrl": "Remote (WAN) access: {0}",
```

Comment:
Rename `Remote (WAN) access` to be consistent. Something like `Remote (WAN) address` sounds right.

Reply:
Yea this does remove it, but just the frontend HTML element - cleaning up this stuff resulted in a bunch of JS breakage.

Reply:
I renamed the string just to be consistent, even if we don't use it.